### PR TITLE
Further updates to QUICK_START.md

### PR DESCRIPTION
**Pull request by Janik-Haag**
See #200

### CODE REVIEW

**Comment by Janik-Haag** [comment]
ping @blitz for review

File: docs/QUICK_START.md [hunk 1]
```diff
@@ -241,7 +233,8 @@ entry in the systemd-boot boot menu. Once you are in the BIOS menu:
 1. Select the "Security" tab.
 2. Select the "Secure Boot" entry.
 3. Set "Secure Boot" to enabled.
-4. Select "Reset to Setup Mode".
+4. Select the "Reset to Setup Mode".
```
Comment (blitz) [inline docs/QUICK_START.md:236]: This change looks accidental?

Comment (Janik-Haag) [reply docs/QUICK_START.md]: Yes sorry, fixed it.

**Review by nikstur — APPROVED** [review]
Looks good. Thank you!

**Pull request merged**